gradle/libs.versions.toml

```diff
@@ -0,0 +1,27 @@
+[versions]
+# this is our own version we deploy
+baseVersion =  "1.29.2"
+
+#versions
+kotlinApi = "1.7"
+kotlin = "1.7.10"
+kotlinJVMtarget = "1.8"
```

Comment:
I think the JVM target does not belong here (rather in gradle.properties or in the build script), but in any case its spelling should be fixed:

```suggestion
kotlinJvmTarget = "1.8"
```

Reply:
Ok